reposcan/exporter.py

```diff
@@ -194,6 +195,62 @@ def dump_errata(self, dump):
+        errataid2pkgs = {}
+        with self._named_cursor() as cursor:
+            cursor.execute("""SELECT pkg_errata.errata_id,
+                                     package_name.name,
+                                     evr.epoch,
+                                     evr.version,
+                                     evr.release,
+                                     arch.name
+                                FROM pkg_errata
+                                JOIN package ON package.id = pkg_errata.pkg_id
+                                JOIN package_name ON package_name.id = package.name_id
+                                JOIN evr ON evr.id = package.evr_id
+                                JOIN arch ON arch.id = package.arch_id
+                           """)
+            for errata_id, p_name, p_epoch, p_ver, p_rel, a_name in cursor:
+                packagename = join_packagename(p_name, p_epoch, p_ver, p_rel, a_name)
+                errataid2pkgs.setdefault(errata_id, []).append(packagename)
+        errataid2bzs = {}
+        errataid2refs = {}
+        with self._named_cursor() as cursor:
+            cursor.execute("SELECT errata_id, type, name FROM errata_refs")
+            for errata_id, ref_type, ref_name in cursor:
+                if ref_type == 'bugzilla':
+                    errataid2bzs.setdefault(errata_id, []).append(ref_name)
+                else:
+                    errataid2refs.setdefault(errata_id, []).append(ref_name)
+        # Now pull all the data together for the dump
+        with self._named_cursor() as cursor:
+            cursor.execute("""SELECT errata.id, errata.name, synopsis, summary,
+                                     errata_type.name, errata_severity.name,
+                                     description, solution, issued, updated
+                                FROM errata
+                                JOIN errata_type ON errata_type_id = errata_type.id
+                                JOIN errata_severity ON severity_id = errata_severity.id
+                           """)
+            for errata_id, e_name, synopsis, summary, e_type, e_severity, \
+                description, solution, issued, updated in cursor:
+                url = "https://access.redhat.com/errata/%s" % e_name
+                dump["errata_detail:%s" % e_name] = (synopsis, summary, e_type,
+                                                     e_severity, description,
+                                                     solution, issued, updated,
+                                                     errataid2cves.get(errata_id, []),
+                                                     errataid2pkgs.get(errata_id, []),
+                                                     errataid2bzs.get(errata_id, []),
+                                                     errataid2refs.get(errata_id, []),
+                                                     url)
+
```

Comment:
I think this service should not export details about every errata found in DB into cache but only security. All selects should have something like `where errata_id in %s` (see other selects) to filter these relevant errata.

Reply:
I included all errata in the dump because I thought we had discussed previously having a flag that would return all errata, not just security ones.

However, I do see that the default behavior is to only return security errata and I did not implement that filter in the default case as I should have.  I'll update the PR to do that.